auslib/db.py

```diff
@@ -644,6 +644,7 @@ def __init__(self, db, dialect, metadata, baseTable):
                 newcol.primary_key = False
             else:
                 newcol.nullable = True
+                newcol.unique = None
```

Comment:
An explanation of why None is used here instead of False would be good - it's not obvious.